adapters/repos/db/vector/cuvs/config.go

```diff
@@ -0,0 +1,63 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package cuvs_index
+
+import (
+	cuvs "github.com/rapidsai/cuvs/go"
+	"github.com/rapidsai/cuvs/go/cagra"
+	"github.com/sirupsen/logrus"
+	"github.com/weaviate/weaviate/entities/errorcompounder"
+	schemaConfig "github.com/weaviate/weaviate/entities/schema/config"
+)
+
+type Config struct {
+	ID               string
+	TargetVector     string
+	Logger           logrus.FieldLogger
+	RootPath         string
+	DistanceMetric   cuvs.Distance
```

Comment:
Can we map to our standard distance functions for compatibility reasons?

Reply:
cuVS only supports L2 distance at the moment, so I just removed it for now